src/main/java/org/cascadebot/cascadebot/data/objects/GuildSettingsMusic.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2020 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package org.cascadebot.cascadebot.data.objects;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.cascadebot.cascadebot.commandmeta.Module;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@SettingsContainer(module = Module.MUSIC)
+@Getter
+@Setter
+@NoArgsConstructor(access = AccessLevel.PUBLIC)
```

Comment:
There's no need to make a public no-args constructor if you do not have any other constructors. It is included into Java by default. This can be removed safely :)